### ISSUE TITLE
Feature request: detect if user is using a custom node

### ISSUE THREAD

**Issue by BlakeBrown** [Berry-Pool/nami-wallet#324]
Omg I'm seeing all the new issues opened on this repo 🙏 hope your team is doing okay!

We have a small feature request, we'd like to be able to detect if the user has a custom node set using Nami so we can decide which set of nodes to use when they submit a transaction on jpg.store.

**Comment by alessandrokonrad** [comment]
Are you currently not using the `submitTx` endpoint from Nami? The submitTx endpoint does automatically handle that. So in case of Nami it's either blockfrost or the custom node if set.

**Comment by BlakeBrown** [comment]
That's correct @alessandrokonrad we're currently using our own set of nodes rather than blockfrost.

We'd like to be able to detect if the user has a custom node set so we can do the following:
1. Try the user's custom node
2. If no response, try our own custom nodes
3. If no response, try blockfrost
